src/OpenSage.Game/Graphics/Shaders/ParticleShaderResources.cs

```diff
@@ -77,7 +81,7 @@ public Pipeline GetCachedPipeline(ParticleSystemShader shader)
                 new ResourceSetDescription(
                     _particleResourceLayout,
                     renderItemConstantsBufferVS,
-                    texture,
+                    texture != null ? texture : _solidWhiteTexture,
```

Comment:
Particle Systems on BFME don't have to have a texture as it seems (atleast the **ParticleTexture** is empty)